generators/export-jdl/index.js

```diff
@@ -17,13 +17,14 @@
  * limitations under the License.
  */
 const chalk = require('chalk');
+const jhCore = require('jhipster-core');
 const BaseGenerator = require('../generator-base');
 const statistics = require('../statistics');
 
 module.exports = class extends BaseGenerator {
     constructor(args, opts) {
         super(args, opts);
-        this.baseName = this.config.get('baseName');
+        this.baseName = this.config.get('baseName') !== undefined ? this.config.get('baseName') : 'jhipster-jdl';
```

Comment:
`this.baseName = this.baseName || 'jhipster-jdl'`
Also, not sure about the default value

Reply:
I used the same default value used by [exportToJdl](https://github.com/jhipster/jhipster-core/blob/master/lib/export/jdl_exporter.js)

Reply:
I think this should be in a constant. My favorite default file name would be `jhipster.jdl` but it's just a personal preference.